### PR TITLE
Blogpost: XSF sign open letter to EC

### PR DESCRIPTION
**Pull request by Echolon**
As agreed in the recent Board meeting

### CODE REVIEW

File: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+date: 2024-01-05
+author: XMPP Standards Foundation
+categories: ["XSF Organisational", "FOSDEM"]
+---
+
+As currently many other organisations doing, the XMPP Standards Foundation (XSF) has decided to also sign the Open Letter to the European Commission initiated by [Petites Singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/) and translated to English by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter)
+
+# The European Union must keep funding free software
+
+Initially published by [petites singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/). English translation provided by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter).
```
Comment (cal0pteryx) [inline content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md:12]: Repetition of above

File: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+categories: ["XSF Organisational", "FOSDEM"]
+---
+
+As currently many other organisations doing, the XMPP Standards Foundation (XSF) has decided to also sign the Open Letter to the European Commission initiated by [Petites Singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/) and translated to English by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter)
+
+# The European Union must keep funding free software
+
+Initially published by [petites singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/). English translation provided by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter). 
+
+If you want to sign the letter, please publish the letter on your website and complete the [table below](https://pad.public.cat/lettre-NCP-NGI).
```
Comment (cal0pteryx) [inline content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md:14]: There is no "table below"

File: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+In this perspective, we urge you to claim for preserving the NGI programme as part of the 2025 funding programme.
+
+[^1]: As defined by Horizon Europe, widening Member States are Bulgaria, Croatia, Cyprus, the Czech Republic, Estonia, Greece, Hungary, Latvia, Lituania, Malta, Poland, Portugal, Romania, Slovakia and Slovenia. Widening associated countries (under condition of an association agreement) include Albania, Armenia, Bosnia, Feroe Islands, Georgia, Kosovo, Moldavia, Montenegro, Morocco, North Macedonia, Serbia, Tunisia, Turkey and Ukraine. Widening overseas regions are : Guadeloupe, French Guyana, Martinique, Reunion Island, Mayotte, Saint-Martin, The Azores, Madeira, the Canary Islands.
+
+## Statement by the XMPP Standards Foundation
+
+The XMPP Standards Foundation supports the initiative as well as the statements made. In the past, many [XMPP projects](https://ngi.eu/?s=xmpp) have been funded by the European Union to enhance realtime infrastructe with our standard but also drive innovation we as Community are building for more than two decades and is well established in the European digital landscape.
```
Comment (cal0pteryx) [inline content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md:46]:
```suggestion
The XMPP Standards Foundation supports this initiative as well as the statements made.
In the past, many [XMPP projects](https://ngi.eu/?s=xmpp) have been funded by NGI programmes to enhance realtime infrastructure using established XMPP standards.
These funds significantly helped drive innovation in a community with two decades of history in the European digital landscape.
```

File: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Below name some of the current project that also show project that go beyond user applications and are also part of EU strategic initiatives:
```
Comment (cal0pteryx) [inline content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md:48]:
```suggestion
These are some of the funded projects:
```

File: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+---
+title: The XSF signs Open Letter to the European Commission
+date: 2024-01-05
+author: XMPP Standards Foundation
+categories: ["XSF Organisational", "FOSDEM"]
```
Comment (cal0pteryx) [inline content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md:5]:
```suggestion
categories: ["XSF Organisational"]
```

**Review by cal0pteryx — CHANGES_REQUESTED** [review]

File: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md (hunk 2 above)
Comment (Echolon) [reply content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md:14]: Thanks, but I will ignore it as it is not from us.

**Pull request merged**